core/src/main/java/nl/nn/adapterframework/extensions/rekenbox/RekenboxLineReader.java

```diff
@@ -1,5 +1,5 @@
 /*
-   Copyright 2013 Nationale-Nederlanden
+   Copyright 2002, 2013 Nationale-Nederlanden, 2023 WeAreFrank!
```

Comment:
Original files written in 2002 according to JavaDoc comment, so I decided to add that to the Copyright notice. 😉